docs/networking/simpleimpl.md

```diff
@@ -10,16 +10,15 @@ First you need to create your `SimpleChannel` object. We recommend that you do t
 
 ```java
 private static final String PROTOCOL_VERSION = "1";
-public static final SimpleChannel INSTANCE = NetworkRegistry.newSimpleChannel(
-  new ResourceLocation("mymodid", "main"),
-  () -> PROTOCOL_VERSION,
-  PROTOCOL_VERSION::equals,
-  PROTOCOL_VERSION::equals
-);
+public static final SimpleChannel INSTANCE = ChannelBuilder.named("mymodid","main"))
+            .clientAcceptedVersions(PROTOCOL_VERSION::equals)
+            .serverAcceptedVersions(PROTOCOL_VERSION::equals)
+            .acceptedVersions(PROTOCOL_VERSION::equals)
+            .optional().simpleChannel();
```

Comment:
Optional should not be used so sparingly. You shouldn't be allowed to connect to the server if the channel isn't there unless the mod does not require server/client communication. Even then, you should choose the side that it makes sense to be optional for.